input/include/NovelRT/Input/InputAction.hpp

```diff
@@ -4,10 +4,6 @@
 #ifndef NOVELRT_INPUT_INPUTACTION_H
 #define NOVELRT_INPUT_INPUTACTION_H
 
-#ifndef NOVELRT_INPUT_H
-#error NovelRT does not support including types explicitly by default. Please include Input.h instead for the Input namespace subset.
-#endif
-
```

Comment:
If we are still relying on a centralised header as per your last comment, we should probably revert this

Reply:
Resolved - i have banished the centralised header to the shadow realm.